### DISCUSSION TITLE
Options / Hints For Variables Selection

### DISCUSSION THREAD

**Discussion by JasonTStanley** [grafana/grafana#55871] [Dashboards, Panels & Visualizations]
Hello all, having used grafana for many dashboards, using Variables to filter Data with lots of fields it would be very helpful to see if there are documents / entries that would result from selecting that option, rather than just having no data available. 

For example,
![image](https://user-images.githubusercontent.com/29510305/192629232-245ccad6-359c-4e7f-8f59-7e52b57dbb80.png)

If i apply some filters such that only Production Systems will match. There could be some indication that given the rest of the filters there are no Development Systems that match the current filters.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
